docs/zh-CN/sql-manual/sql-functions/window-functions/WINDOW-FUNCTION-NTILE.md

```diff
@@ -0,0 +1,43 @@
+---
+{
+    "title": "WINDOW-FUNCTION-NTILE",
+    "language": "zh-CN"
+}
+---
+
+<!--  Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.  See the NOTICE file distributed with this work for additional information regarding copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License. -->
+
+## WINDOW FUNCTION NTILE
+### description
+
+将排序分区中的行划分为特定数量的组。从每个组分配一个从一开始的桶号。对于每一行， NTILE() 函数返回一个桶号，表示行所属的组。
+
+```sql
+NTILE() OVER(partition_by_clause order_by_clause)
+```
+
+### example
+
+```sql
+select x, y, ntile(2) over(partition by x order by y) as rank from int_t;
+
+| x | y    | rank     |
+|---|------|----------|
+| 1 | 1    | 1        |
+| 1 | 2    | 1        |
+| 1 | 2    | 2        |
+| 2 | 1    | 1        |
+| 2 | 2    | 1        |
+| 2 | 3    | 2        |
+| 3 | 1    | 1        |
+| 3 | 1    | 1        |
+| 3 | 2    | 3        |
```

Comment:
error, different with english result. should be `2`

Reply:
I fixed it.